hack/local-up-cluster.sh

```diff
@@ -432,7 +436,7 @@ function start_apiserver {
       fi
       RUNTIME_CONFIG+="scheduling.k8s.io/v1alpha1=true"
     fi
-    
+
```

Comment:
If it's possible, please, avoid unrelated changes.

Reply:
Just update by the hand, cannot bear the empty space in my vim setup.